samples/sensor/fxos8700/README.rst

```diff
@@ -1,38 +1,87 @@
 .. _fxos8700:
 
-FXOS8700 Accelerometer/Magnetometer Sensor
-##########################################
+FXOS8700 Driver Sample
```

Comment:
Please change the commit message. You're not adding a new README.rst, you're changing the existing one to also work with MMA8451, MMA8652, and MMA8653 accelerometers.